system-tests/management-api/management-api-test-runner/src/test/java/org/eclipse/edc/test/e2e/managementapi/ContractDefinitionApiEndToEndTest.java

```diff
@@ -59,6 +61,65 @@ void queryContractDefinitions_noQuerySpec() {
         assertThat(criteria).hasSize(2);
     }
 
+    private JsonObject createSingleFilterQuery(String leftOperand, String operator, String rightOperand) {
```

Comment:
private methods should stay below the public/protected/package private ones

Reply:
Done